sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session_receiver.py

```diff
@@ -115,7 +116,7 @@ def from_connection_string(
         # type: (str, Any) -> ServiceBusSessionReceiver
         """Create a ServiceBusSessionReceiver from a connection string.
 
-        :param conn_str: The connection string of a Service Bus.
+        :param str conn_str: The connection string of a Service Bus.
```

Comment:
nitpick: in some places it's
```
:param conn_str: The connection string of a Service Bus.
:type conn_str: str
```
while here it's 
```:param str conn_str: The connection string of a Service Bus.```

we'd better be consistent.

Reply:
This was actually one of the other questions I posed for @annatisch , whether my understanding for our decision tree in using those was accurate; namely:  prefer the terse syntax (inline) but if this would become problematic (e.g. complex type, very long signature) utilize next-line.

Reply:
conclusion from email: if the former generates line length or formatting issues, fall back to the latter.  Don't know if that means we should just always use the latter, I'd be tempted to for consistency but it's just _oh so verbose._  Any strong feelings?

Reply:
(my personal preference would be the "inline" as much as possible, for the sake of reducing numbers of line🤣)